apps/vs-code-designer/src/app/commands/generateDeploymentScripts/azureScriptWizard.ts

```diff
@@ -0,0 +1,251 @@
+  folderPath?: string;
+}
+
+/**
+ * Creates an instance of the Azure Wizard for the Azure Script Wizard.
+ * @param wizardContext - The wizard context.
+ * @param projectPath - The path of the project.
+ * @returns An instance of the Azure Wizard.
+ */
+// Your existing function for creating the Azure Wizard
+export function createAzureWizard(wizardContext: IAzureScriptWizard, projectPath: string): AzureWizard<IAzureScriptWizard> {
+  // Existing prompt steps
+  const promptSteps = [
+    new ConfigureInitialLogicAppStep(),
+    new setLogicappName(),
+    new setStorageAccountName(),
+    new setAppPlantName(),
+    new SourceControlPathListStep(),
+  ];
+  const executeSteps: AzureWizardExecuteStep<IAzureScriptWizard>[] = [
+    new SaveAzureContext(projectPath),
+    // other steps
+  ];
+
+  if (!isMultiRootWorkspace) {
+    wizardContext.isValidWorkspace = false;
+  } else {
+    wizardContext.isValidWorkspace = true;
+  }
+
+  // Create the Azure Wizard with the modified steps
+  return new AzureWizard(wizardContext, {
+    promptSteps,
+    executeSteps,
+  });
+}
+
+export class SourceControlPathListStep extends AzureWizardPromptStep<IAzureScriptWizard> {
+  public hideStepCount = true;
+
+  private async createDeploymentFolder(rootDir: string): Promise<string> {
+    const deploymentsDir = path.join(rootDir, 'Deployment');
+    if (!fs.existsSync(deploymentsDir)) {
+      fs.mkdirSync(deploymentsDir);
+    }
+    return deploymentsDir;
+  }
+
+  public static setSourceControlPath(context: Partial<IAzureScriptWizard>, artifactsPath: string): void {
+    context.sourceControlPath = artifactsPath;
+    context.workspacePath = (context.workspaceFolder && context.workspaceFolder.uri.fsPath) || context.customWorkspaceFolderPath;
+    if (context.workspaceFolder) {
+      context.openBehavior = OpenBehavior.alreadyOpen;
+    }
+  }
+
+  public async prompt(context: IAzureScriptWizard): Promise<void> {
+    const workspaceFileUri = vscode.workspace.workspaceFile;
+    let rootDir = context.customWorkspaceFolderPath;
+    //change the root directory if unable to locate workspace file
+    if (!workspaceFileUri) {
+      rootDir = context.folderPath;
+    } else {
+      rootDir = path.dirname(workspaceFileUri.fsPath);
+    }
+
+    let deploymentFolderExists = false;
+    if (rootDir) {
+      deploymentFolderExists = fs.existsSync(path.join(rootDir, 'Deployment'));
+    }
+
+    const deploymentLabel = deploymentFolderExists ? 'Deployment folder in current workspace' : 'New deployment folder';
+
+    const placeHolder = 'Select the folder to store your deployment artifacts';
+    const options: vscode.QuickPickItem[] = [
+      {
+        label: deploymentLabel,
+        description: deploymentFolderExists
+          ? 'Uses the existing deployment folder in the current workspace.'
+          : 'Creates a new deployment folder in the current workspace.',
+      },
+      {
+        label: 'Choose a different Folder',
```

Comment:
"Folder" shouldn't be capitalized if it's not the first word:
 
```suggestion
        label: 'Choose a different folder.',
```